### PR TITLE
(#73) Add contrib, issue, and PR guidelines

### LINKED ISSUE #73: CONTRIBUTING.md
Type up a `CONTRIBUTING.md` file

### PR DESCRIPTION
**Pull request by llorllale**
Closes #73

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+* [Link to the GitHub issue it solves (if any)]
+* Make sure the following build targets pass:
```
Comment (rolsonquadras) [inline .github/PULL_REQUEST_TEMPLATE.md]: For me, this looks more like a guidelines to raise than a PR template.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+**What I'm trying to do**
+
+[Clearly describe what you're trying to do.]
```
Comment (rolsonquadras) [inline .github/ISSUE_TEMPLATE.md]: This looks like a Bug report template. We might need a [Feature Request](https://help.github.com/en/articles/about-issue-and-pull-request-templates#issue-templates) template as well.

Comment (llorllale) [reply .github/ISSUE_TEMPLATE.md]: Note: we don't have the permissions required to use the template builder, so we're stuck with the old way of adding templates

Comment (llorllale) [reply .github/ISSUE_TEMPLATE.md]: @rolsonquadras is there any specific change you're requesting for this file?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (llorllale) [reply .github/PULL_REQUEST_TEMPLATE.md]: @rolsonquadras any specific change you'd like to see?

Comment (rolsonquadras) [reply .github/PULL_REQUEST_TEMPLATE.md]: @llorllale looks like you have updated this with [new patch](https://github.com/hyperledger/aries-framework-go/compare/f4423945b781038c7655a19c0dbe820aee0c6944..151000517d2512bebf2d96f64b648a8e9a36f796#diff-195a635ad245ded9076330e31134bd80R4).

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (rolsonquadras) [reply .github/ISSUE_TEMPLATE.md]: As you mentioned about the [permission issue](https://github.com/hyperledger/aries-framework-go/pull/75#discussion_r311748819), not sure we can do much. I'm ok with [patch](https://github.com/hyperledger/aries-framework-go/compare/f4423945b781038c7655a19c0dbe820aee0c6944..151000517d2512bebf2d96f64b648a8e9a36f796#diff-1a1c3dd142f76a5fad803a0c52839881R2) mentioning about feature request.

Comment (llorllale) [reply .github/ISSUE_TEMPLATE.md]: We're calling feature requests "proposals". Is there anything else you'd like to be added?

**Pull request merged**